<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>885</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea_2">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-518</y>
        <width>633</width>
        <height>1838</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_productImg">
           <property name="text">
            <string>Product Image</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QFrame" name="frame">
           <property name="minimumSize">
            <size>
             <width>400</width>
             <height>400</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="0">
             <widget class="QLabel" name="product_image">
              <property name="text">
               <string>Product Image</string>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_productUpc">
           <property name="text">
            <string>UPC Image</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QFrame" name="frame_2">
           <property name="minimumSize">
            <size>
             <width>400</width>
             <height>400</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="upc_image">
              <property name="text">
               <string>UPC Image</string>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_generated_barcode">
           <property name="text">
            <string>Generated UPC</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QFrame" name="frame_3">
           <property name="minimumSize">
            <size>
             <width>400</width>
             <height>400</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="0" column="0">
             <widget class="QLabel" name="generated_upc">
              <property name="text">
               <string>Generated UPC</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_id">
           <property name="text">
            <string>ID</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QSpinBox" name="ID">
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="maximum">
            <number>999999999</number>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_name">
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="name">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_upc">
           <property name="text">
            <string>UPC</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLineEdit" name="upc">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_homecode">
           <property name="text">
            <string>Home Code</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QLineEdit" name="homecode">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_case_count">
           <property name="text">
            <string>Case Count</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QSpinBox" name="case_count">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="label_comment">
           <property name="text">
            <string>Comment</string>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QScrollArea" name="scrollArea">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>250</height>
            </size>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>508</width>
              <height>246</height>
             </rect>
            </property>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="0" column="0">
              <layout class="QGridLayout" name="gridLayout_3">
               <item row="0" column="0">
                <widget class="QTextEdit" name="comment">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>200</height>
                  </size>
                 </property>
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item row="10" column="0">
          <widget class="QPushButton" name="brand_btn">
           <property name="text">
            <string>Brand</string>
           </property>
          </widget>
         </item>
         <item row="11" column="0">
          <widget class="QPushButton" name="vendor_btn">
           <property name="text">
            <string>Vendor</string>
           </property>
          </widget>
         </item>
         <item row="12" column="0">
          <widget class="QPushButton" name="manufacturer_btn">
           <property name="text">
            <string>Manufacturer</string>
           </property>
          </widget>
         </item>
         <item row="13" column="0">
          <widget class="QPushButton" name="department_btn">
           <property name="text">
            <string>Department</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="close">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
